Hearthstone Deck Tracker/Utility/Helper.cs

```diff
@@ -54,8 +54,8 @@ public static class Helper
 		public static readonly Dictionary<string, string> LanguageDict = new Dictionary<string, string>
 		{
 			{"English", "enUS"},
-			{"Chinese (China)", "zhCN"},
-			{"Chinese (Taiwan)", "zhTW"},
+			{"简体中文", "zhCN"},
+			{"繁體中文", "zhTW"},
```

Comment:
The dropdown should be translated yes, but I don't know if this particular dict should be. @azeier ?